### DISCUSSION TITLE
Tutorial 6: ViewSets & Routers should mention that format_suffix_patterns is included in the DefaultRouter

### DISCUSSION THREAD

**Discussion by paatre** [encode/django-rest-framework#8990] [Ideas & Suggestions]
Hello everyone,

I recently completed the official tutorial and found it to be an excellent resource for getting started with DRF more confidently. I've had a little bit of experience with DRF before but I hadn't quite fully connected the dots yet about the core aspects of DRF. I think that's in the past now. However, I noticed that there is an aspect related to format suffix patterns that could benefit from further clarification in the tutorial.

In Tutorial 6: ViewSets & Routers, the use of the `DefaultRouter` class is introduced. When the `DefaultRouter` is introduced, previous usage of `format_suffix_patterns` is dropped as well and it's a bit confusing. It is worth mentioning that the `DefaultRouter` includes the automatic handling of format suffix patterns. This behavior might not be immediately evident to newcomers following the tutorial if it isn't mentioned. At least for me it wasn't. I needed to check from the source code myself that the format suffix patterns are still supported.

TLDR: I suggest adding a brief mention or explanation to the tutorial about the inclusion of `format_suffix_patterns` in the `DefaultRouter`.

Thank you for considering my suggestion, and I look forward to any feedback or further discussion on this topic.

Best regards,
Teemu Viikeri
